src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -739,6 +767,10 @@ void ArchiveBuilder::relocate_metaspaceobj_embedded_pointers() {
   log_info(cds)("Relocating embedded pointers in core regions ... ");
   relocate_embedded_pointers(&_rw_src_objs);
   relocate_embedded_pointers(&_ro_src_objs);
+  log_info(cds)("Relocating %zu pointers, %zu tagged, %zu nulled",
```

Comment:
`log_info(aot)` if it is Leyden related.

Reply:
This more like a generic cds message. I'll leave this one as is.